indra/llrender/llrender.cpp

```diff
@@ -879,6 +879,11 @@ void LLRender::init(bool needs_vertex_buffer)
     // necessary for reflection maps
     glEnable(GL_TEXTURE_CUBE_MAP_SEAMLESS);
 
+    if (glGenVertexArrays == nullptr)
```

Comment:
`&glGenVertexArrays`
otherwise xcode refuses to build this.

Reply:
Sorry, missed a commit. it's supposed to be 'windows only'

Reply:
Or picket it from a wrong branch...